doc/user/layouts/shortcodes/fnlist.html

```diff
@@ -46,6 +52,18 @@ <h3 id="{{ lower .type | replaceRE "[^a-zA-Z0-9]+" "-" }}-func">
       {{ if .side_effecting }}
         <br><br><b>Note:</b> This function is <a href="#side-effecting-functions">side-effecting</a>.
       {{ end }}
+
+      {{ $versionAdded := index . "version-added" }}
+      {{ if $versionAdded }}
+        {{ $releasePage := index $releasedVersions $versionAdded }}
+        {{ if not $releasePage.Params.released }}
+          <br><br>
+          <b>Rolling out: </b> This function is currently rolling out
+          in <a href="{{ printf "/releases/%s" $versionAdded | relURL }}"><strong>{{$versionAdded}}</strong></a>.
+          It may not be available in your region yet.
+          Rollout is scheduled to complete by <strong>{{dateFormat "January 2, 2006" $releasePage.Params.date}}</strong>.
```

Comment:
```suggestion
          <b>Not released: </b> This function will be released
          in <a href="{{ printf "/releases/%s" $versionAdded | relURL }}"><strong>{{$versionAdded}}</strong></a>.
          It may not be available in your region yet.
          The release is scheduled to complete by <strong>{{dateFormat "January 2, 2006" $releasePage.Params.date}}</strong>.
```